<launch>
    <node pkg="view_changer" type="view_changer_node.py" name="view_changer_node" output="screen">
        <param name="topic_name" value="/rviz/camera_placement" />
        <param name="views_path" value="$(find view_changer)/config/views.json" />
    </node>
</launch>